🔧 Update pyproject&pre-commit config

ARYAN-NIKNEZHAD opened the pull request:
### Description
- Add bandit for check python security
- Add bandit hook in pre-commit
- Update lock file
- Update requirements files

ARYAN-NIKNEZHAD committed:
:wrench: chore: Update pyproject&pre-commit config
- Add bandit for check python security
- Add bandit hook in pre-commmit
- Update lock file
- Update requirements files

sepehr-akbarzadeh commented on pyproject.toml:
```diff
@@ -75,6 +76,25 @@ norecursedirs = [
+progress = true
+reports = true
+output_format = "screen"
+output_file = "bandit_report.txt"
```
here you have bandit_report.txt but not inside the .gitignore file.

ARYAN-NIKNEZHAD committed:
:ok_hand: fix: fix code review changes

ARYAN-NIKNEZHAD committed:
:wrench: chore: Update pyproject.toml config
- Add django debug-toolbar and migration-linter
- update lock file
- Update requirements-dev.txt file
